src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -201,7 +201,7 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
       <table className={styles.dextable}>
         <thead>
           <tr>
-            <td> Currency</td>
+            <td style={{ textAlign: 'left' }}> Currency</td>
```

Comment:
@abhisheknaiidu can we move this to css file, we should try avoiding the use of inline 😅 🙇🏻